docs/docs/deploying-to-21yunbox.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Deploying to 21YunBox
+---
+
+[21YunBox](https://www.21yunbox.com) provides blazing fast Chinese CDN, Continuous Deployment, one click HTTPS and [other services like ..... ](https://www.21yunbox.com/docs/v2/index.html), providing developers a hassle-free solution to launch their web projects in China.
```

Comment:
Hey can you add the other features. as a replacement for "....... "

Reply:
Oh, didn't see that earlier.

Sure. replace "..." with "managed databases and backend web services".

Should I recommit it?